scripts/zkai_tablero_hybrid_zkml_boundary_gate.py

```
@@ -0,0 +1,1066 @@
+        "source_artifacts": [
+            source_descriptor(MINIMAL_BENCHMARK, sources["minimal"], raw_sources["minimal"]),
+            source_descriptor(GKR_BASELINE, sources["gkr"], raw_sources["gkr"]),
+            source_descriptor(JOLT_ATLAS_COMPARISON, sources["jolt"], raw_sources["jolt"]),
+            source_descriptor(JSTPROVE_STATEMENT_ENVELOPE, sources["jstprove"], raw_sources["jstprove"]),
+        ],
+        "non_claims": list(NON_CLAIMS),
+        "validation_commands": list(VALIDATION_COMMANDS),
+    }
+
+
+def validate_binding_object(value: Any, label: str) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)